scripts/generate_dump

```diff
@@ -536,17 +711,59 @@ collect_mellanox() {
+    if [ -d /usr/share/sonic/device/${platform}/${hwsku} ]; then
+        # copy all the files in the HWSKU directory
+        pushd /usr/share/sonic/device/${platform}/${hwsku} > /dev/null
+        for file in $(find . -maxdepth 1 -type f); do
+            save_file ${file} sai false
+        done
+        popd > /dev/null
+        # config.bcm - copy the one with chip common properties merged
+        for file in $(find /var/run/docker-syncd -type f -name "*.bcm"); do
+            save_file ${file} sai false
+        done
+        # sai.profile - copy the final sai.profile generated in docker-syncd
+        if [ -f /var/run/docker-syncd/sai.profile ]; then
+            save_file /var/run/docker-syncd/sai.profile sai false
+        fi
+    else
+        echo "'/usr/share/sonic/device/${platform}/${hwsku}' does not exist" > /tmp/error
+        save_file /tmp/error sai false
+    fi
+
+    save_bcmcmd "bcmcmd -t5 version" "broadcom.version"
+    save_bcmcmd "bcmcmd -t5 soc" "broadcom.soc"
+    save_bcmcmd "bcmcmd -t5 ps" "broadcom.ps"
     save_cmd "cat /proc/bcm/knet/debug" "broadcom.knet.debug"
     save_cmd "cat /proc/bcm/knet/dma" "broadcom.knet.dma"
-    save_cmd "cat /proc/bcm/knet/dstats" "broadcom.knet.dstats"
     save_cmd "cat /proc/bcm/knet/link" "broadcom.knet.link"
     save_cmd "cat /proc/bcm/knet/rate" "broadcom.knet.rate"
-    save_cmd "cat /proc/bcm/knet/stats" "broadcom.knet.stats"
-    save_cmd "bcmcmd \"l3 nat_ingress show\"" "broadcom.nat.ingress"
-    save_cmd "bcmcmd \"l3 nat_egress show\"" "broadcom.nat.egress"
+    save_bcmcmd "bcmcmd \"l3 nat_ingress show\"" "broadcom.nat.ingress"
+    save_bcmcmd "bcmcmd \"l3 nat_egress show\"" "broadcom.nat.egress"
+    save_bcmcmd "bcmcmd \"ipmc table show\"" "broadcom.ipmc"
+    save_bcmcmd "bcmcmd \"multicast show\"" "broadcom.multicast"
+    save_bcmcmd "bcmcmd \"conf show\"" "conf.summary"
+    save_bcmcmd "bcmcmd \"fp show\"" "fp.summary"
+    save_bcmcmd "bcmcmd \"pvlan show\"" "pvlan.summary"
+    save_bcmcmd "bcmcmd \"l2 show\"" "l2.summary"
+    save_bcmcmd "bcmcmd \"l3 intf show\"" "l3.intf.summary"
+    save_bcmcmd "bcmcmd \"l3 defip show\"" "l3.defip.summary"
+    save_bcmcmd "bcmcmd \"l3 l3table show\"" "l3.l3table.summary"
+    save_bcmcmd "bcmcmd \"l3 egress show\"" "l3.egress.summary"
+    save_bcmcmd "bcmcmd \"l3 ecmp egress show\"" "l3.ecmp.egress.summary"
+    save_bcmcmd "bcmcmd \"l3 multipath show\"" "l3.multipath.summary"
+    save_bcmcmd "bcmcmd \"l3 ip6host show\"" "l3.ip6host.summary"
+    save_bcmcmd "bcmcmd \"l3 ip6route show\"" "l3.ip6route.summary"
+    save_bcmcmd "bcmcmd \"mc show\"" "multicast.summary"
+    save_bcmcmd "bcmcmd \"cstat *\"" "cstat.summary"
+    save_bcmcmd "bcmcmd \"mirror show\"" "mirror.summary"
+    save_bcmcmd "bcmcmd \"mirror dest show\"" "mirror.dest.summary"
+    save_bcmcmd "bcmcmd \"port *\"" "port.summary"
+    save_bcmcmd "bcmcmd \"d chg my_station_tcam\"" "mystation.tcam.summary"
```

Comment:
all the 'bcmcmd' commands must be run for all ASICs on multi ASIC platforms. The format is 
```
bcmcmd -n <asic ID> <options..>
```
Please also update the ones that already existed and not updated.

Reply:
Done